yml/hsc.yaml

```diff
@@ -1008,6 +1008,126 @@ tables:
+    description: HSM higher order moments 12. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_12
+    "@id": "#Object.g_hsm_momentsPsf_12"
+    datatype: double
+    description: HSM higher order PSF moments 12. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_03
+    "@id": "#Object.g_hsm_moments_03"
+    datatype: double
+    description: HSM higher order moments 03. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_03
+    "@id": "#Object.g_hsm_momentsPsf_03"
+    datatype: double
+    description: HSM higher order PSF moments 03. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_40
+    "@id": "#Object.g_hsm_moments_40"
+    datatype: double
+    description: HSM higher order moments 40. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_40
+    "@id": "#Object.g_hsm_momentsPsf_40"
+    datatype: double
+    description: HSM higher order PSF moments 40. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_31
+    "@id": "#Object.g_hsm_moments_31"
+    datatype: double
+    description: HSM higher order moments 31. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_31
+    "@id": "#Object.g_hsm_momentsPsf_31"
+    datatype: double
+    description: HSM higher order PSF moments 31. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_22
+    "@id": "#Object.g_hsm_moments_22"
+    datatype: double
+    description: HSM higher order moments 22. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_22
+    "@id": "#Object.g_hsm_momentsPsf_22"
+    datatype: double
+    description: HSM higher order PSF moments 22. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_13
+    "@id": "#Object.g_hsm_moments_13"
+    datatype: double
+    description: HSM higher order moments 13. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_13
+    "@id": "#Object.g_hsm_momentsPsf_13"
+    datatype: double
+    description: HSM higher order PSF moments 13. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_04
+    "@id": "#Object.g_hsm_moments_04"
+    datatype: double
+    description: HSM higher order moments 04. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_momentsPsf_04
+    "@id": "#Object.g_hsm_momentsPsf_04"
+    datatype: double
+    description: HSM higher order PSF moments 04. Measured on g-band.
+    mysql:datatype: DOUBLE
+    fits:tunit:
+  - name: g_hsm_moments_flag
+    "@id": "#Object.g_hsm_moments_flag"
+    datatype: boolean
+    description: General failure flag, to be used in conjunction with g_i_flag. Measured on g-band.
```

Comment:
For my own curiousity: what's `g_i_flag`?  Is `i` here generic for "moment"?

Reply:
Yeah, it follows from `ixx` notation. My guess is that `i` here is from moment of *i*nertia